contracts/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Prettier Solidity formatting applied to v0.7 and above.
 - ERC677ReceiverInterface added in v0.8.
+- `ArbitrumSequenceStatusRecorderInterface` and ``ArbitrumSequenceStatusRecorder` added in v0.8.
```

Comment:
@chainlink/contract package 0.2.3 has already been cut. Can we move these additions and changes to a new unreleased section please?

Reply:
I moved these to a new `Unreleased` section and removed `Unreleased` from `0.2.3` 👍